src/test/java/uk/gov/justice/digital/service/DataStorageServiceTest.java

```diff
@@ -288,21 +288,6 @@ public void shouldRetryAppendDistinctAndSucceedEventually() throws DataStorageEx
         verify(mockDeltaMergeBuilder, times(2)).execute();
     }
 
-    @Test
-    public void shouldRetryUpsertRecordsAndSucceedEventually() throws DataStorageRetriesExhaustedException {
```

Comment:
Both upsert and delete now use the same merge operation leaving us with two duplicate tests which have now been removed.